source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/index.tsx

```diff
@@ -164,10 +179,10 @@ class Notifications extends React.Component<IProps> {
                     }
 
                     return (
-                      <div
-                        className={`notification-item ${isViewed ? 'seen' : 'not-seen'}`}
-                        key={index}
-                      >
+                      <div className={`notification-item ${isViewed ? 'seen' : 'not-seen'}`} key={index} onClick={e => {
```

Comment:
ternaries should not be used in the template portion of a component.
assigning to a variable before the template provides more semantic context. :)
Also, I've recently read that `key` shouldn't be assigned to the loop index, but something that ties the render to the item being rendered, maybe like the `id` of the notification